backend/routers/issues.py

```
@@ -228,6 +240,8 @@ async def create_issue(
         # Invalidate cache so new issue appears
         try:
             recent_issues_cache.clear()
+            if user_email:
+                user_issues_cache.clear()
         except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)